eng/pipelines/templates/jobs/ci.mgmt.yml

```diff
@@ -15,6 +15,15 @@ jobs:
     steps:
       #- script: "echo $(system.pullrequest.pullrequestnumber), https://github.com/$(build.repository.id), https://github.com/$(build.repository.ID)"
       - template: /eng/pipelines/templates/steps/install-dotnet.yml
+      - task: UseDotNet@2
+        condition: and(succeeded()) # We still need .NET Core 3.1 for .NET Track 1 SDK
```

Comment:
condition succeeded() is redundant but doesn't hurt anything either.

Reply:
Looks like this condition does need to be removed because the and operator requires 2 arguments https://dev.azure.com/azure-sdk/public/_build/results?buildId=2677828&view=results

Reply:
Good catch, removed.